heka/meta/heka.yml

```diff
@@ -68,6 +68,16 @@ metric_collector:
       module_dir: /usr/share/lma_collector/common;/usr/share/heka/lua_modules
       preserve_data: false
       message_matcher: "Type == 'heka.all-report'"
+    influxdb_accumulator:
+      engine: sandbox
+      module_file: /usr/share/lma_collector/filters/influxdb_accumulator.lua
+      module_dir: /usr/share/lma_collector/common;/usr/share/heka/lua_modules
+      preserve_data: false
+      message_matcher: "Fields[aggregator] == NIL && Type =~ /metric$/"
+      ticker_interval: 1
+      config:
+        tag_fields: "deployment_id environment_label hostname tenant_id user_id"
```

Comment:
Eventually deployment_id and environment_label need to be added in the model and injected into the tag_fields.lua template.
I think you can drop the other tags because they should be added by the corresponding decoders/filters whenever they are present.